index.html

```diff
@@ -1913,6 +1919,64 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    [=People=] should be provided relevant information when data is accessed or when permission is
+    requested.
+  </span>
+</div>
+
+Transparency is a necessary, but insufficient, condition for [=consent=]. Relevant information
+includes who may be accessing information, what information may be accessed (including the potential
+inferences or combinations of such data) and how information may be used. For transparency to be
+meaningful to people, it must be contextual.
+
+<div class="note">
+  In designing new Web features that may involve permissions, consider whether a permission is
+  needed and how to make that permission meaningful [[?ADDING-PERMISSIONS]].
+
+  Past workshops have explored the needs for better permissions on the Web:
+  <ul>
+    <li><a href="https://www.w3.org/Privacy/permissions-ws-2022/report">2022 W3C Workshop on
+    Permissions</a></li>
+    <li><a href="https://www.w3.org/Privacy/permissions-ws-2018/report.html">2018 W3C Workshop on
+    Permissions and User Consent</a></li>
+    <li><a href="https://www.w3.org/2014/07/permissions/minutes.html">2014 Next steps on trust and
+    permissions for Web applications</a></li>
+  </ul>
+</div>
+
+<div class="practice">
+  <span class="practicelab" id="transparency-plain-language-machine-readable">
+    Information about privacy-relevant practices should be provided in both easily accessible plain
+    language form and in machine-readable form.  
+  </span>
+</div>
+
+Machine-readable presentation of privacy-relevant practices is necessary for [=user agents=] to be
+able to help [=people=] make general decisions, rather than relying falsely on the idea that
+[=people=] can or want to read documentation before every visit to a web site. Machine-readable
+presentation also facilitates <a href="#collective">collective governance</a> by making it more
+feasible for researchers and regulators to discover, document, and analyze data collection and
+processing in ways that may be harmful.
```

Comment:
I don't think we want to support researchers analysing data in ways that may be harmful :-D

```suggestion
processing to identify cases in which it may be harmful.
```